ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -0,0 +1,247 @@
+The data runs via the *p* input port into the workflow action and is output via both the *p0* and the *p1* output port.
+
+[comment]: <> (ticket in arbeit: mehr als zwei output port sollen hinzukommen)
+
+### Configuration   
+
+This core action has no further configuration settings.
+
+
+
+## Execute PHP code
+
+![Execute PHP code](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/ExecutePHPCode.png "[Execute PHP code]")
+
+The *Execute PHP code* core action is used to execute a custom PHP code defined in the configuration.
+As the complete functionality of the PHP code is accessible, this core action enables complete variability in programming and is therefore often used when more complex actions which require more logic need be executed within a workflow step.   
+Depending on which ports are connected to the places and which ports are defined in the PHP code, the data runs via the *in0* to *in9* input ports into the workflow action and is output via the *out0* to *out9* output ports.   
+It is also possible to include a static input via the unconnected input ports. These inputs must be defined in the *Static inputs* section in the settings side bar, see [Transitions](../Overview/04_WorkflowProcessElements#transitions).
+
+> [Info] It is recommended to give the action a name that describes what the PHP code does.
+
+### Configuration   
+
+- *PHP Code*   
+  Click this field to display the code editor and enter the desired PHP code. There are no restrictions in the PHP code. Just remember to include the input ports whose date should be read using **$in0** for the *in0* input port, **$in1** for the *in1* input port, etc. as well as to return the output via the output ports using the *return [output1, output2];* return array. The first value within the array is output via the *out0* output port, the second value via the *out1* output port, etc.  
+
+  ![PHP code editor](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/PHPCodeEditor.png "[PHP code editor]")
+
+
+
+## Split by criterion
+
+![Split by criterion](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/SplitByCriterion.png "[Split by criterion]")
+
+The *Split by criterion* core action is used to compare the input value with a certain criterion and if it matches, a different branch is followed up than if it differs.   
+The data runs via the *in* input port into the workflow action and is either output via the *match* output port if the input value matches the criterion to be compared with, or via the *noMatch* output port if the input value does not match the criterion to be compared with.   
+The criterion to be compared with is defined in the configuration.
+
```

Comment:
"criterion to be compared with is defined" vgl. "with which criterion the input value is comprared"
Was ist mit was verglichen?